src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1078,6 +1110,11 @@ private async Task<DebugStore> RuntimeReady(SessionId sessionId, CancellationTok
                 Log("verbose", $"Failed to clear breakpoints due to {clear_result}");
             }
 
+            if (context.PauseOnCaught && context.PauseOnUncaught)
+                await SendMonoCommand(sessionId, MonoCommands.SetPauseOnExceptions("all"), token);
+            if (context.PauseOnUncaught)
+                await SendMonoCommand(sessionId, MonoCommands.SetPauseOnExceptions("uncaught"), token);
```

Comment:
Won't this send both commands when `context.PauseOnCaught && context.PauseOnUncaught` is true?

Reply:
yes, you are right!